bcipy/signal/model/offline_analysis.py

```diff
@@ -297,161 +297,167 @@ def analyze_gaze(
     )
     ''' Trigger_timing includes PROMPT and excludes FIXATION '''
 
+    # Extract the inquiries dictionary with keys as target symbols and values as inquiry windows:
+    symbol_set = alphabet()
```

Comment:
This should be an argument that perhaps defaults to alphabet.

Reply:
I updated the Gaze Reshaper to have `alphabet()` as the default argument. The reason I also have this as a dictionary `symbol_set` is for the iterative use cases.